server/document.js

```diff
@@ -97,11 +97,14 @@ export class NextScript extends Component {
 
   render () {
```

Comment:
If you add a link tag with the attribute "preload" in the head, it gives the browser a hint that we are going to be loading some scripts later on in the document. This could possibly give us a slight boost in Time To Interactive.  See this for some background. https://www.smashingmagazine.com/2016/02/preload-what-is-it-good-for/ 

So in the _document, for the initial components (errorComponent and the current route bundle) you could render <link rel="preload" href="/_next/pages/<page>.js"> and possibly have them evaluated sooner. But we should profile it and see if it helps. Does that make more sense?